### PR TITLE
Fix for color palette demo  [Issue #120]

### PR DESCRIPTION
**Pull request by rox163**
# patch
# CHANGELOG
- Fixed issue #120

### CODE REVIEW

File: tests/dummy/config/environment.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ module.exports = function (environment) {
     modulePrefix: 'dummy',
     podModulePrefix: 'dummy/pods',
     environment: environment,
-    rootURL: '/',
+    baseURL: '/',
```
Comment (rox163) [inline tests/dummy/config/environment.js:7]: the LTS prep work in progress will make this update properly.

File: tests/dummy/app/pods/components/demo-palette/component.js [hunk 2]
```diff
@@ -13,7 +13,9 @@ function rgb2hex (rgb) {
 
   rgb = rgb.match(/^rgb\((\d+),\s*(\d+),\s*(\d+)\)$/)
 
-  return `#${hex(rgb[1])}${hex(rgb[2])}${hex(rgb[3])}`
+  return hex(rgb[1]) === '0'
```
Comment (sandersky) [inline tests/dummy/app/pods/components/demo-palette/component.js]: Would we run the risk of `rgb[2]` or `rgb[3]` being `0` as well? Instead should it be something like:

``` js
const hex = rgb.slice(1)
  .map((value) => value === '0' ? '00' : `${hex(value)}`)
  .join('')

return `#${hex}`
```

Comment (rox163) [reply tests/dummy/app/pods/components/demo-palette/component.js]: Doh! yes! Just so happens none of our current colour palette has that.

**Comment by sandersky** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/209/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

**Pull request merged**